match/lib/match/nuke.rb

```diff
@@ -240,6 +263,11 @@ def delete_files!
 
     # The kind of certificate we're interested in
     def certificate_type(type)
+      case type.to_sym
+      when :mac_installer_distribution
+        return [Spaceship.certificate.mac_installer_distribution]
+      end
```

Comment:
this method should be just a regular o' `case` instead of this hash-lookup-with-raise.
```ruby
case type.to_sym
when :mac_installer_distribution
  [Spaceship.certificate.mac_installer_distribution]
when :distribution
  [Spaceship.certificate.production, Spaceship.certificate.apple_distribution]
...
else
  raise "Unknown certificate type `#{type}`
end
```

Reply:
Smart 💡